src/aui/framemanager.cpp

```diff
@@ -647,7 +648,11 @@ wxAuiManager::~wxAuiManager()
 
     delete m_art;
 }
-
+void wxAuiManager::OnSysColourChanged(wxSysColourChangedEvent& event) {
```

Comment:
Minor style: please separate functions with blank lines and put `{` on its own line for consistency.

Reply:
Will fix...